common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -1233,6 +1287,10 @@ def submit_problem(self, data, override_time=False):
         # render problem into HTML
         html = self.get_problem_html(encapsulate=False, submit_notification=True)
 
+        # Withhold success indicator if hiding correctness
+        if not self.correctness_available():
```

Comment:
Never mind-- I think you didn't want to change the event being logged (which I agree with!). Please leave as-is.